### PR TITLE
added new trello items and api references

### PR DESCRIPTION
**Pull request by MHolmes219**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -8,6 +8,9 @@ authors = []
 python = "^3.7"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
+arrow = "^1.2.1"
+python-dateutil = "^2.8.2"
+requests = "^2.26.0"
```
Comment (BenRamchandani) [inline pyproject.toml:13]: You should commit the resulting changes to `poetry.lock` too.

File: todo_app/data/trello_items.py [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+import dateutil.parser
+from datetime import datetime
+
+def get_auth_params():
+    return { 'key': app.config['API_KEY'], 'token': app.config['API_TOKEN'] }
+
+def build_url(endpoint):
+    return app.config['BASE_URL'] + endpoint
+
+def build_params(params = {}):
```
Comment (BenRamchandani) [inline todo_app/data/trello_items.py]: Style point: private methods in Python that aren't intended to be used outside of the current file normally have their name prefixed with `_`, e.g. `_build_params`.

File: todo_app/data/trello_items.py [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+    params = build_params({ 'idList': list['id'] })
+    url = build_url('/cards/%s' % cardId)
+
+    response = requests.put(url, params = params)
+    card = response.json()
+
+    return card
+
+
+def update_card_status(cardId, list):
```
Comment (BenRamchandani) [inline todo_app/data/trello_items.py]: Minor, but this function name is a little unclear, maybe `close_and_move_card`?

**Review by BenRamchandani — APPROVED** [review]
This all looks great, nice work.

I've left some optional comments below, feel free to merge when you're ready.

File: todo_app/data/trello_items.py (hunk 2 above)
Comment (MHolmes219) [reply todo_app/data/trello_items.py]: Thanks Ben. I have updated to reference double underscore for the private methods.

File: todo_app/data/trello_items.py (hunk 3 above)
Comment (MHolmes219) [reply todo_app/data/trello_items.py]: Updated as requested.

**Pull request merged**